src/components/migrationToV3.stories.mdx

```diff
@@ -85,6 +92,216 @@ const MyApp = () => {
 };
 ```
 
+### Meeting Provider param change
+
+Meeting provider no longer takes the following props:
+- `MeetingSessionConfiguration` props:
+  - `VideoDownlinkBandwidthPolicy`
+  - `VideoUplinkBandwidthPolicy`
+  - `simulcastEnabled`
+  - `reconnectTimeoutMs`
+  - `keepLastFrameWhenPaused`
+- Other props:
+  - `logLevel`
+  - `postLogConfig`
+  - `logger`
+  - `enableWebAudio`
+  - `activeSpeakerPolicy`
+
+These properties can rather be updated by passing the `MeetingSessionConfiguration` object or by passing `options` param to `meetingManager.join()` method. 
+
+You can learn more about the `meetingManager.join()` method here: [MeetingManager](/docs/sdk-providers-meetingmanager--page).
+
+#### Examples
+
+##### VideoDownlinkBandwidthPolicy
+
+The following example uses `VideoDownlinkBandwidthPolicy` property as an example to showcase the change.
+If your code looks like this:
+```jsx
+import { useMeetingManager } from 'amazon-chime-sdk-component-library-react';
+
+const MyApp = () => {
+  // MyVideoDownlinkBandwidthPolicy is a custom class implementing VideoDownlinkBandwidthPolicy interface from amazon-chime-sdk-js.
+  const videoDownlinkBandwidthPolicy = new MyVideoDownlinkBandwidthPolicy();
+  const meetingConfig = {
+    videoDownlinkBandwidthPolicy,
+  };
+
+  const meetingManager = useMeetingManager();
+  await meetingManger.join({
+    data.Meeting,
+    data.Attendee,
+  });
+
+  return (
+  <MeetingProvider {...meetingConfig}>
+    <MyApp />
+  </MeetingProvider>
+  );
+};
+```
+
+change it to like this:
+```jsx
+import { useMeetingManager } from 'amazon-chime-sdk-component-library-react';
+import { MeetingSessionConfiguration } from 'amazon-chime-sdk-js';
+
+const MyApp = () => {
+  // MyVideoDownlinkBandwidthPolicy is a custom class implementing VideoDownlinkBandwidthPolicy interface from amazon-chime-sdk-js.
+  const videoDownlinkBandwidthPolicy = new MyVideoDownlinkBandwidthPolicy();
+
+  const meetingSessionConfiguration = new MeetingSessionConfiguration(data.Meeting, data.Attendee);
+  meetingSessionConfiguration.videoDownlinkBandwidthPolicy = videoDownlinkBandwidthPolicy;
+
+  const meetingManager = useMeetingManager();
+  await meetingManager.join(
+    meetingSessionConfiguration,
+  );
+
+  return(
+    <MeetingProvider>
+      <MyApp />
+    </MeetingProvider>
+  );
+};
+```
+
+##### simulcastEnabled
+
+One thing to note is that the `MeetingProvider` props might not have the same mapping with `MeetingSessionConfiguration` properties.
+For example, `simulcastEnabled` in `MeetingProvider` and the property in `MeetingSessionConfiguration` is different. The following code example should help remove any confusion.
```

Comment:
I don't think we need to mention this. As `simulcastEnabled` and `enableSimulcastForUnifiedPlanChromiumBasedBrowsers` have different naming. They are not supposed to be the same thing.

It confuse me a little bit. I would prefer removing these or make it more clear.